openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -0,0 +1,212 @@
+## Instantiating a Workspace from Torch Template
+To instantiate a workspace from the torch_cnn_mnist template, you can use the fx workspace create command. This allows you to quickly set up a new workspace based on a predefined configuration and template.
+1. Ensure the necessary dependencies are installed.
+"""
+pip install virtualenv
+mkdir ~/openfl-quickstart
+virtualenv ~/openfl-quickstart/venv
+source ~/openfl-quickstart/venv/bin/activate
+pip install openfl
+"""
+2. Creating the Workspace Folder
+
+"""
+cd ~/openfl-quickstart
+fx workspace create --template torch_template --prefix fl_workspace
+cd ~/openfl-quickstart/fl_workspace
+"""
```

Comment:
I suggest you use Markdown code blocks for the shell commands. Otherwise they don't render very well.
See [here](https://github.com/securefederatedai/openfl/blob/d4bf63d127e571ca6d4c182fe2df247e7eea220a/openfl-workspace/torch_cnn_mnist/README.md#instantiating-a-workspace-from-torch-template).

Reply:
resolved in be37d1c1e35fee2cce4ee8907547c861e28e6d8f